README.md

```diff
@@ -87,6 +91,9 @@ to master).
 
 To do that, just connect as root on the puppetmaster node and run apply-in-review-patches.sh.
 You may have some manual merges to do ... :/
+```
```

Comment:
The cmd must be run with the reset argument before in order to deactivate run_all.sh. Please add it and after I merge this PR. Thanks Tristan !